### ISSUE TITLE
DL3041: false positive

### ISSUE THREAD

**Issue by yellowhat** [hadolint/hadolint#1136]
Hi,
after updating to `ghcr.io/hadolint/hadolint:v2.14.0-alpine`, I get:

```
./kind/Dockerfile:12 DL3041 warning: Specify version with `dnf install -y <package>-<version>`.
```

The `Dockerfile`:

```Dockerfile
FROM quay.io/podman/stable:v5.6.1

ENV KIND_EXPERIMENTAL_PROVIDER podman
# Set kubeconfig custom location to avoid overlap with
# serviceaccount token when running in a pod
ENV KUBECONFIG=/tmp/kubeconfig

SHELL ["/bin/bash", "-o", "pipefail", "-c"]

# renovate: datasource=repology depName=fedora_42/git versioning=loose
ARG GIT_VER=2.51.0-2.fc42
RUN dnf install -y \
        "git-core-${GIT_VER}" \
 && dnf clean all
```

Using `ghcr.io/hadolint/hadolint:v2.13.1-alpine` it works as expected

**Comment by m-ildefons** [comment]
Fixed in v2.15.0
